### ISSUE TITLE
Controlling scroll position

### ISSUE THREAD

**Issue by brettz9** [atom/bookmarks#96]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

I find it awkward that the scroll position for bookmarks is at the top for the first bookmark and toward the button for subsequent ones. It would be nice to have bookmarks be consistently positioned (and ideally also offering the ability to either center or specify the number of lines of context from the top or bottom).

**Comment by bryonwinger** [comment]
+1

**Comment by bernk** [comment]
Agreed. This is mentioned here too: https://github.com/atom/bookmarks/issues/50

**Comment by Daijobou** [comment]
+1
